dashboard/api.py

```diff
@@ -29,7 +29,8 @@
 from profiles.api import get_social_auth
 
 # maximum number of exam attempts per payment
-ATTEMPTS_PER_PAID_RUN = 2
+ATTEMPTS_PER_PAID_RUN_OLD = 2  # the number of attempts the user gets for payment before the first date
```

Comment:
Is it possible to rename this? What makes it different from `ATTEMPTS_PER_PAID_RUN`?

Reply:
I named like this because we are intending to remove it completely after `second date` passes. I thought it would be easy to distinguish.